db_structure.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE Products (
     id VARCHAR(10) PRIMARY KEY,
-    brand_id VARCHAR(255),
+    brand_id INT,
```

Comment:
How are you planning to migrate the data in production? Do you plan to just delete everything and then add back the items?

Reply:
I will create a script to update the data type of the brand_id: 
```
ALTER TABLE brands 
ALTER COLUMN brand_id TYPE INT;
```